README.md

```diff
@@ -209,16 +209,16 @@ The best way to install this extension is to follow the instructions below. Howe
 - Use this to control how faint(weakly visible) or strongly visible you want the edge(un-emphasized) part of words to appear.
 
 #### Saccades colors
-- Use this to select a means of emphasization using colors.
+- Use this to select a means of creating emphasis using colors.
 
 #### Saccades styles
-- Use this to select a means of emphasization using bold variations or underline variations. 
+- Use this to select a means of creating emphasis using bold variations or underline variations. 
 
 #### Line height buttons
-- Use these buttons to increase or decrease line height to strain and improve the confort of reading.
+- Use these buttons to increase or decrease line height to strain and improve the comfort of reading.
 
 #### Always on/off button
-- Use this button to controls the default behaviour which is if words on pages are or aren't emphasized when loaded by default.
+- Use this button to controls the default behavior which is if words on pages are or aren't emphasized when loaded by default.
```

Comment:
British spelling here, sharp eye really, thank you

Reply:
My 1st PR. Happy to do some more or make some adjustments as needed